connect/src/com/telenor/connect/ConnectSdkProfile.java

```diff
@@ -124,4 +128,12 @@ private String applyUseStaginfOnEndpoint(String endpoint) {
         }
         return endpoint.replace("connect.telenordigital.com", "connect.staging.telenordigital.com");
     }
+
+    @Override
+    public void validateTokens(ConnectTokensTO tokens, Date serverTimestamp) {
+        super.validateTokens(tokens, serverTimestamp);
+        Validator.notNullOrEmpty(tokens.getScope(), "scope");
+        Validator.notNull(tokens.getExpiresIn(), "expires_in");
+        Validator.notNullOrEmpty(tokens.getRefreshToken(), "refresh_token");
```

Comment:
Connect will return a response without refresh token in case there is something wrong with the db, the access token will still be valid in that case.